lib/workers/pr/changelog/release-notes.ts

```diff
@@ -249,21 +288,29 @@ export async function addReleaseNotes(
     const cacheKey = getCacheKey(v.version);
     releaseNotes = await renovateCache.get(cacheNamespace, cacheKey);
     if (!releaseNotes) {
-      releaseNotes = await getReleaseNotesMd(
+      releaseNotes = await getReleaseNotes(
```

Comment:
What is the reason for changing the order of this logic? Before it checked markdown first, now it checks non-markdown?

Reply:
Extraction of the Changelog.md wasn't working well for any of the dependencies tested in gitlab. They were all using the Releases object, which seemed more fit for purpose and easier to extract. Might be good to test against a github hosted dependency with only Markdown notes, for backwards compatibility.

Reply:
Might not be so easy to find one and doesn't seem like the most backwards compatible. Would it be OK if I just skip the markdown for Gitlab and put the order as it was before?

Reply:
I suspect that quite a few *self-hosted* GitLab users would use files like `CHANGELOG.md`. If that suspicion is correct then it wouldn't be long until someone tries the feature and expects their markdown files to be found. Even if you can't find examples on gitlab.com, is there any problem you see with keeping the lookup logic there?

Reply:
Laziness, TBH, I need to make some changes for the Markdown to work with gitlab. The releases was more straight forward and seemed sufficient for our use case, but you have a point there, don't think there's a way around it. Will put the MD first and do the amendments, thanks for the feedback.

Reply:
That went better than expected, I don't remember why I was having so much trouble when I decided to alter the order, but looks better now.